src/doc/man/generated_txt/cargo-add.txt

```diff
@@ -96,6 +96,12 @@ OPTIONS
            Mark the dependency as required
            <https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies>.
 
+       --public
+           Mark the dependency as public.
```

Comment:
Should we flag this as "nightly only", and have a link to unstable doc?

(same thing also applies to `cargo add -h/--help`?)

Reply:
Somehow I thought of this and forgot it all in the time it took for me to look at this and approve it...

Reply:
Ok，I will add the "nightly only" and the link latter.